app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -277,6 +280,17 @@ public void onClick(View v) {
     }
 
     @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == 1 && resultCode == RESULT_OK) {
```

Comment:
Look at MainActivity.java

    public void openSpotActivity() {
        Intent intent = new Intent(MainActivity.this, StudentSpotActivity.class);
        startActivityForResult(intent, 1);
    }

It is an arbitrary number that the user decides on